source/includes/kubernetes/_k8_ingresses.md

```diff
@@ -1,6 +1,9 @@
 ### Ingresses V1Beta1
 
+Ingress v1beta1 has been deprecated in this API as of 26.7.0 following the removal of support by kubernetes. Please use the <a href="#kubernetes-ingresses-v1">ingress v1 api instead</a>.
+
 <!-------------------- LIST INGRESSES V1Beta1 -------------------->
+<!-- 
```

Comment:
what is the purpose of this? are we commenting out the contents of v1beta1?

Reply:
yeah if you recall we removed that code so if you call any of those APIs you will simply get this:
<img width="572" alt="image" src="https://user-images.githubusercontent.com/19736641/191264038-9297aa88-b6f2-4147-9164-7925c7050266.png">